backend/src/main/java/ca/bc/gov/backendstartapi/enums/ActivityEnum.java

```diff
@@ -1,12 +1,13 @@
 package ca.bc.gov.backendstartapi.enums;
 
+import com.fasterxml.jackson.annotation.JsonValue;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import java.util.Optional;
 import lombok.Getter;
 import lombok.ToString;
 
 /** This enumeration contains all possible activities. */
 @Getter
-@ToString
 public enum ActivityEnum {
```

Comment:
We came to the conclusion that it makes more sense to store the information here in the front-end and for this enum to have no properties.

Reply:
Yup. Commit sent.